apis/goharbor.io/v1alpha2/harborcluster_types.go

```diff
@@ -130,6 +131,12 @@ type MinIOSpec struct {
 	// Number of persistent volumes that will be attached per server
 	// +kubebuilder:validation:Minimum:=1
 	VolumesPerServer int32 `json:"volumesPerServer"`
+	// ExternalCertSecret allows a user to specify custom CA certificate, and private key. This is
+	// used for enabling TLS support on MinIO Pods.
+	// +kubebuilder:validation:Optional
+	ExternalCertSecret *minio.LocalCertificateReference `json:"externalCertSecret,omitempty"`
+	// +kubebuilder:validation:Optional
```

Comment:
I think `Host` should be required when it needs to be exposed.

Reply:
In the case that the user does not need the exposed `Host`, the `Host` can be left unset, so maybe it should be `Optional`.

Reply:
I think we should consider together with the above comment.